common/errors.go

```diff
@@ -0,0 +1,13 @@
+package common
+
+import (
+	"deliverables/common/constants"
+)
+
+func HandleError(message string, err error) (bool, constants.PageData) {
+	if err != nil {
+		return true, constants.PageData{Message: message, Status: "Error"}
+	} else {
```

Comment:
The else is not needed because you have a return in the if.
```go 
if err != nil {
	return true, constants.PageData{Message: message, Status: "Error"}
}
return false, constants.PageData{}
```